NativeAuthSampleApp/ProtectedAPIViewController.swift

```diff
@@ -181,6 +208,11 @@ class ProtectedAPIViewController: UIViewController {
         
         task.resume()
     }
+    
+    func arrayContainsSubarray(array: [String], subarray: [String]) -> Bool {
```

Comment:
This should be in an extension

Reply:
In a regular application, this should go to an extension. But the sample app doesn't have any extension, all code is laid out in the view controllers, so novice developers can see all in one place. We are not using standard architectures in this project, by design.